contrib/fontforge/cidmap/CMakeLists.txt

```diff
@@ -14,15 +17,34 @@ if(ENABLE_MAINTAINER_TOOLS)
   target_link_libraries(mapkorean PRIVATE fontforge)
 endif()
 
-install(
-  FILES
+set(
+  CIDMAPS
     Adobe-CNS1-6.cidmap
     Adobe-GB1-5.cidmap
+    Adobe-Identity-0.cidmap
     Adobe-Japan1-5.cidmap
     Adobe-Japan1-6.cidmap
     Adobe-Japan1-7.cidmap
     Adobe-Japan2-0.cidmap
     Adobe-Korea1-2.cidmap
-  DESTINATION ${CMAKE_INSTALL_DATADIR}/fontforge/cidmap
+)
+
+if (NOT INSTALL_IDENTITY_CIDMAP)
+  list(REMOVE_ITEM CIDMAPS Adobe-Identity-0.cidmap)
+endif()
+
+install(
+  FILES ${CIDMAPS}
+  DESTINATION "${CIDMAP_DIR}"
   COMPONENT cidmaps
 )
+
+if(NOT INSTALL_IDENTITY_CIDMAP)
```

Comment:
I don't see why this is special cased like so, especially when none of the other installed files do this.

If you don't want it there do a clean install.

Reply:
Oh, just thought it'd help fix bugs on some people's machines. I don't mind removing it.

Reply:
> Nevertheless I need it sometimes. Further, the way it was deleted would keep it around due to the way ninja install works.

Please confirm whether you think this explanation is good enough. I don't think that the directory will get deleted even by a clean install especially if it contains other user-provided CIDMaps, so I don't think a clean install is really a good solution..